.travis.yml

```diff
@@ -9,6 +9,7 @@ addons:
 install:
   # Needed to run setup.py
   - pip install cython numpy
+  - pip install git+https://github.com/ModellingWebLab/cellmlmanip.git@more-trig-functions
```

Comment:
This will need to change back before merging :)

Reply:
Does that mean I need to ask maurice to do a cellmlmanip release before we can merge this?

Reply:
Yes please @MauriceHendrix  :)